features/step_definitions/helper_services.rb

```diff
@@ -161,9 +161,11 @@
     # So take the second one since this one can be implemented currently
     ###
     stats = {}
-    step %Q/I run the :run client command with:/, table(%{
-      | name  | ldapserver                                       |
-      | image | quay.io/openshifttest/ldap:multiarch |
+    step 'I obtain test data file "authorization/ldapserver.yaml"'
+    step %Q/the step should succeed/
```

Comment:
Step `the step should succeed` seems not needed for step `I obtain test data file ...`

Reply:
Done